src/App.js

```diff
@@ -14,8 +17,14 @@ class App extends React.Component {
     return (
       <div className="App">
         <div className="App-body" id="App-body">
-          <QuoteComponent />
-          <Calculator />
+          <BrowserRouter>
+          <Navbar />
+          <Routes>
+            <Route path='/home' element={<Home />} />
```

Comment:
- [ ] When I run the app in the browser, the route is `/` without 'home' so no page is opened. then I have to click on either link to get that specific page opened. I think when the app runs first time, the home page must get opened and displayed on the screen. for that purpose, you can write this line like this `<Route path='/' element={<Home />} />` and for it to be implemented perfectly you also need to change something in Navbar.js file. kindly have a look there for guidelines.